index.html

```diff
@@ -1,11 +1,32 @@
 <!doctype html>
 <html>
   <head>
-    <title>Donut Shops</title>
+    <title>DeLorean Dounts</title>
   </head>
   <body>
-    <h1>Donut Shops</h1>
+    <h1>DeLorean Donuts</h1>
    <table id="donut-shops"></table>
+   <form id="donut-form">
+     <label for="location-name">Loc</label>
+      <input name="location-name" type="text">
```

Comment:
You could, but don't have to in this case, change the `type` of the input tag to `number`, which would provide you an `int` data type when calling the `.valule` in your JS.